samples/customchecks/monitoring3rdparty/Version_3_3/3rdPartySystem/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
```

Comment:
minor point. can we make this sample a little lighter by using a httplistener to represent the backend instead of taking 4 dependencies?